<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>proppindlg</class>
 <widget class="QDialog" name="proppindlg">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>370</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Pin Property</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout">
   <item row="3" column="0">
    <widget class="QGroupBox" name="groupBox1">
     <property name="title">
      <string>PinType</string>
     </property>
     <layout class="QGridLayout">
      <item row="1" column="2">
       <widget class="QRadioButton" name="radioZero">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff/>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="radioCk">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff/>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="radioS">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff/>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="radioNeg">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff/>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radioNegCk">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff/>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioNorm">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff/>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <layout class="QVBoxLayout">
     <item>
      <widget class="QPushButton" name="buttonOK">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="checkHide">
     <property name="text">
      <string>Hide Number</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout">
     <item>
      <layout class="QHBoxLayout">
       <item>
        <widget class="QLabel" name="textLabel2">
         <property name="text">
          <string>Number</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editNumber">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>255</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <item>
        <widget class="QLabel" name="textLabel2_2">
         <property name="text">
          <string>Block</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBlock"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="textLabel1">
       <property name="text">
        <string>Name</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editName">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>255</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>editName</tabstop>
  <tabstop>editNumber</tabstop>
  <tabstop>spinBlock</tabstop>
  <tabstop>checkHide</tabstop>
  <tabstop>radioNorm</tabstop>
  <tabstop>radioCk</tabstop>
  <tabstop>radioNeg</tabstop>
  <tabstop>radioNegCk</tabstop>
  <tabstop>radioS</tabstop>
  <tabstop>radioZero</tabstop>
  <tabstop>buttonOK</tabstop>
  <tabstop>buttonCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>proppindlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>379</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
